diagrams/src/classdiagram.pu

```diff
@@ -0,0 +1,79 @@
+
+@startuml
```

Comment:
Make sure to add names to the diagrams (needed when exporting to images).

`@startuml {The name of the diagram}`